README.md

```diff
@@ -68,15 +65,54 @@ between these mechanisms, but an override is deemed the simplest proposal initia
 (3) avoids the need to specify the fetch option conditions that the integrity would have to apply to for other assets. It may be possible to relax this constraint
 in future that integrity can apply to other assets as well, but that would require more carefully defining the associated fetch conditions for which it would apply.
 
+## Worker Import Maps
+
+> Status: Initial Proposal
+
+### Problem Statement
+
+Import maps only apply to the main application thread, and any workers created via `new Worker(path)` will never share the import map resolutions.
+
+When writing modular applications that rely on import map resolutions, any code running in a worker cannot utilize import maps at all.
+
+### Proposal
+
+The proposal is to allow supporting import maps in workers, with a new `importMap` attribute for the worker instantiation options, where the
+import map for the worker can be provided explicitly:
```

Comment:
Inheriting the import map should be the default? That's what I expected and then went down the rabbit hole...